pkg/localstore/subscription_push_test.go

```diff
@@ -50,9 +52,22 @@ func TestDB_SubscribePush(t *testing.T) {
 			}
 
 			chunks = append(chunks, ch)
+
+			chunkProcessedTimes = append(chunkProcessedTimes, 0)
 		}
 	}
 
+	// REQUIRED_LOCK - chunksMu
```

Comment:
what does the comment mean? that you're supposed to call this under lock? if so - you can just write in the comment `// caller expected to hold lock on chunksMu`